### ISSUE TITLE
[Bug]: typekit Get Url error

### ISSUE THREAD

**Issue by jslee-ncurity** [cssninjaStudio/unplugin-fonts#63]
### What happened?

`
      typekit: {
        id: `${myTypekitId}`,
        defer: false,
        injectTo: 'body',
      }
`

When executing the above code 
GET https://use.typekit.net/myTypekitId.css net::ERR_ABORTED 412 (Precondition Failed)
An error occurs.

In fact, if you run the code provided by Typekit locally, you need to get the css code here at https://use.typekit.net/myTypekitId.js

**to sum up**
https://use.typekit.net/myTypekitId.css (X)
https://use.typekit.net/myTypekitId.js (O)

### Reproduction steps

```bash
1.
2.
3.
...
```
